packages/layout/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@telegraph/layout",
```

Comment:
Moved what was `@telegraph/box` to `@telegraph/layout` to simplify importing these layouts. Also matches chakra's package naming scheme for similar components.

Reply:
Yeah I like this naming a lot!